testutils/src/main/scala/za/co/absa/enceladus/testutils/HelperFunctions.scala

```diff
@@ -133,12 +132,54 @@ object HelperFunctions {
     * Calculates the time it took for passed block of code to execute and finish.
     * @param processToRun Block of code, to be executed
     * @tparam A A type parameter specifying output from processToRun
-    * @return Returns a millisecond difference between start and end time
+    * @return Returns a tuple of milliseconds difference between start and end time
+    *         and return value
     */
-  def calculateTime[A](processToRun: => A): ProcessMeasurement[A] = {
+  def calculateTime[A](processToRun: => A): (Long, A) = {
     val startTime = System.nanoTime()
     val returnValue = processToRun
     val endTime = System.nanoTime()
-    ProcessMeasurement(endTime - startTime, returnValue)
+    val millisecondsPassed = (endTime - startTime) / 1000000
+    (millisecondsPassed, returnValue)
+  }
+
+  /**
+    * Pretty prints elapsed time. If given 91441000 will return "1 day, 1 hour, 24 minutes and 1.00 second"
+    * @param elapsedTime Elapsed time in milliseconds you want to pretty print
+    * @return Returns a string format with human readable time segments
+    */
+  def prettyPrintElapsedTime(elapsedTime: Long): String = {
```

Comment:
The function is awesome!

To be completely perfect it would be cool to have a couple of unit tests for it to ensure it works out time elements correctly.

Reply:
And to have examples of how it prints. I get it. Hopefully, the new ones are ok.